seatunnel-e2e/seatunnel-connector-v2-e2e/connector-jdbc-e2e/connector-jdbc-e2e-part-2/src/test/java/org/apache/seatunnel/connectors/seatunnel/jdbc/JdbcOceanBaseMysqlIT.java

```diff
@@ -68,12 +68,12 @@ public class JdbcOceanBaseMysqlIT extends JdbcOceanBaseITBase {
                                 "mkdir -p /tmp/seatunnel/plugins/Jdbc/lib && cd /tmp/seatunnel/plugins/Jdbc/lib && wget "
                                         + driverUrl()
                                         + " && wget "
-                                        + mysqlDriverUrl());
+                                        + oceanbaseDriverUrl());
                 Assertions.assertEquals(0, extraCommands.getExitCode(), extraCommands.getStderr());
             };
 
-    String mysqlDriverUrl() {
-        return "https://repo1.maven.org/maven2/com/mysql/mysql-connector-j/8.0.32/mysql-connector-j-8.0.32.jar";
```

Comment:
Is it compatible with older versions?

Reply:
Yes, you can use the old version or the new version, and only need oceanbase-client